iOS14UIKitTour/iOS14UIKitTour/API/GithubAPI.swift

```diff
@@ -0,0 +1,17 @@
+
+import Foundation
+import Combine
+
+enum GithubAPI {
+    /// Returns a publisher that performs a GET request.
+    ///
+    /// - Parameters:
+    ///     - endpoint: the GithubEndpoint enum case which maps to the appropriate URL.
+    static func fetch<T: Decodable>(_ endpoint: GitHubEndpoints) -> AnyPublisher<[T], Error> {
+        return URLSession.shared.dataTaskPublisher(for: endpoint.url)
+            .receive(on: RunLoop.main)
+            .map(\.data)
+            .decode(type: [T].self, decoder: JSONDecoder())
+            .eraseToAnyPublisher()
+    }
+}
```

Comment:
😍 This is beautiful. The comments, the solution, everything is chef kiss.

Reply:
Thanks so much! Has no error handling, ship it anyway?

Reply:
Yeah, it's fine for this demo. You can come back to it if you want.